src/docs/product/accounts/sso/azure-sso.mdx

```diff
@@ -57,17 +57,62 @@ description: "Set up Azure Active Directory Single Sign-On (SSO) on Sentry."
     - First Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname`
 
     - Last Name: `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`
-    
+
       For more details about mappings for custom configs, see [The Role of Claims](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
 
    ![Map Identity with provider attributes](azure-map-identity-provider-attributes.png)
-   
+
 1. Sentry will attempt to authenticate and link your account with Azure. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
 
    - You can share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
-   
+
    - Scroll down to the bottom and ensure that "Require SSO" is checked if you want to enforce logging in with Okta.
-   
+
    - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use-cases.
-   
+
    - If you made changes, click "Save Settings" to complete your setup.
+
+## SCIM Integration
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+Sentry users can manage provisioning using Azure with SCIM. You'll need to have Azure SSO set up and configured for your organization already. Sentry supports User and Group provisioning with Azure.
+
+
+### Sentry Configuration
```

Comment:
It seems like this could use a bit of spacing from the above paragraph, but not sure how to do so within the markdown here.

Reply:
yes, this does look a bit squashed. I'm not sure why. Try inserting spaces above the Note to see it renders (I'll add those lines as suggestions).